index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile Version Skeleton</title>
+</head>
+
+<body>
+    <section class="container">
+        <section class="pg-1">
+            <header>
+                
+            </header>
+
+            <div class="logo">
+
+                <h2 class="my-logo"><a href="#" target="_blank" rel="noopener noreferrer">My Logo</a></h2>
```

Comment:
Addressed ✔

Reply:
@mateo951 Sorry about that. I would definitely wait for approval henceforth. Thanks so much.